### ISSUE TITLE
[A11Y] Textarea form field no focusable

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9329]
## Issue Description
It is not possible to focus and add text to the " comments for the editors"  textarea form field. When using a keyboard, the user gets stuck on the "WYSIWYG toolbar" and is not able to add content to the element.

## How to reproduce / Current behavior
1. Create a submission
2. Include requested info on each screen + "continue" until step 4 "For the editors" 
3. See the error when loading the screen (trapped on textarea toolbar)

## What application are you using?
 OJS version 3.4.0

## Stack used 
- Device: Laptop
- OS: MacOS(Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Method: Manual - Keyboard

## Action
1. Implement the focus on the text area instead of the editor toolbar icons.
2. Includes instructions about standard system shortcuts to edit text  e.g.: **bold** = CTRL/COMMAND + B

## Additional information

https://github.com/pkp/pkp-lib/assets/4992902/0b3914bf-c037-44dd-892e-381e1f622ea5

**Comment by Devika008** [comment]
Capturing a detailed response from @israelcefrin about solution proposed: 

We provide users with instructions about how to use keyboard shortcuts with same functionality of the toolbar. Bold, Italic and link creation have well known shortcuts: CTRL+B, CTRL+ I, CTRL + K.

Also, to format any text using keyboard only will require to select the word/expression and move away from the main text area (without using TABm otherwise it will remove the selected text adding a tabbing space).
